### PR TITLE
use fetch instead of jquery

### PR DESCRIPTION
**Pull request by herra**
Based on #35

### CODE REVIEW

File: rt_dashboard/javascript/src/api.js [hunk 1]
```diff
@@ -8,6 +8,23 @@ const requestErrorHandler = error => {
...
 
+/**
+ *
+ * @param {Promise<Response>?} requestPromise
+ * @returns {Promise<any>|null}
+ */
+const executeRequest = async (requestPromise) => {
+  try {
+    const response = await requestPromise
+    if (response.ok) {
```
Comment (groupsky) [inline rt_dashboard/javascript/src/api.js:18]: status code 500 is also `ok`

**Review by groupsky — APPROVED** [review]
looks ok, but could have been split into using `fetch` without depending on #35 and another that converts callbacks to async/await

File: rt_dashboard/javascript/src/api.js (hunk 1 above)
Comment (herra) [reply rt_dashboard/javascript/src/api.js:18]: I used mdn as reference: https://developer.mozilla.org/en-US/docs/Web/API/Response maybe I got the wrong interface.

**Comment by herra** [comment]
> 
> 
> looks ok, but could have been split into using `fetch` without depending on #35 and another that converts callbacks to async/await

I wanted to but it brought too many conflicts. Both PRs change/move pieceses of code in tasks/queues/workers components

File: rt_dashboard/javascript/src/api.js (hunk 1 above)
Comment (groupsky) [reply rt_dashboard/javascript/src/api.js:18]: Sorry, my mistake

File: rt_dashboard/javascript/src/api.js [hunk 2]
```diff
@@ -1,5 +1,3 @@
-import $ from 'jquery'
```
Comment (groupsky) [inline rt_dashboard/javascript/src/api.js:1]: could be removed from `package.json`

**Review by groupsky — COMMENTED** [review]
What's with the empty `toast.js`?

**Pull request merged**